### ISSUE TITLE
[Feature Request] Vertically compress the list of stories.

### ISSUE THREAD

**Issue by g1dd5** [FredJul/Flym#430]
Could Flym show the list of unread/all stories in a more compact way?

My device (a Gemini PDA) works only in landscape mode, and Flym can show only two story titles at once.  If the site logo were shrunk, the font size reduced a little, and/or the vertical spacing reduced, it could probably fit many more, which would be much more useful.

(I guess this could either be a user option, or automatic based on the screen height.)

Flym seems to handle landscape mode very well, other than this issue.  (I open stories in my browser, so I can't speak about how it displays story bodies.)

Finally, some praise: Flym is exactly what I want from a news reader!  I'd given up finding one that didn't require accounts and subscriptions, store my feeds on some company's servers, or try to turn news reading into a poor social network…  So many thanks for it!

**Comment by Noexpert** [comment]
I agree this would be a useful feature, especially for those of us with many feeds.
